.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:10.13
+
+    working_directory: ~/tasit/demo
+
+    steps:
+      - checkout:
+          path: ~/tasit
+
+      - run:
+          name: update-npm
+          command: "sudo npm install -g npm@latest"
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - dependency-cache-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - dependency-cache-
+
+      - run:
+          name: npm install
+          command: npm install
+
+      - save_cache:
+          paths:
+            - ./node_modules
+          key: dependency-cache-{{ checksum "package.json" }}
+
+      # run tests!
+      - run:
+          name: test
+          command: npm test
+
+      - store_artifacts:
```

Comment:
FYI this step and the one after it isn't working well for the Tasit SDK repo, so we should remove it here

Check out the steps below `test` here https://circleci.com/gh/tasitlabs/TasitSDK/23

Reply:
Done.